app/helpers/application_helper.rb

```diff
@@ -1,2 +1,57 @@
 module ApplicationHelper
+  def us_states
+    [
+      ['AK', 'AK'],
+      ['AL', 'AL'],
+      ['AR', 'AR'],
+      ['AZ', 'AZ'],
```

Comment:
I added this helper for the dropdown menu. I wasn't sure of the best way to go about this, so if anyone has any ideas, I would love to hear them!

Reply:
I think we might be able to use a gem to help. Let's look into it during work time this morning!

Reply:
I am a little confused about what this does.

Reply:
@sagemlee I'm using this method in the form so they all appear as options when the user is selecting a state.

Reply:
I may not have fully understood the functionality. This is awesome and maybe implementing a gem is unnecessarily complicated 😅